tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/charts/At-risk_learners.yaml

```diff
@@ -13,18 +13,55 @@ params:
   - name
   - username
   - email
+  - expressionType: SQL
+    label: Course Grade %
+    sqlExpression: round(course_grade*100, 2)
```

Comment:
This one should be updated on DBT, it would simplify translations.

Reply:
Can you explain that? We do the `round()` thing in a lot of places